README.md

```diff
@@ -1,5 +1,6 @@
 # OpenELIS Global 2
```

Comment:
Can you get rid of the format changes in the Read me , i dont see any actual change